README.md

```diff
@@ -35,19 +35,19 @@ ability
 <img align="center" src = "https://github.com/Tejas1510/Athavani/blob/master/assets/Images/heroku.png"></img>
 </p>
 
-## A sneek peek into what we have here🙈 :
+## A sneak peek into what we have here🙈 :
 
 ### Creation of Memory
-<p align="center"><img src="https://github.com/Tejas1510/Athavani/blob/master/assets/creation.gif"></p>
+<p align="center"><img src="https://github.com/mitalirs/Athavani/blob/update-readme/assets/createMemory.gif"></p>
```

Comment:
Change back the username to `Tejas1510` from your name in URL, and also `master` from `update-readme`.

Reply:
yes, sure, actually, I had changed it for the time being so that the gifs would be visible to the reviewer, as the new gifs are not present in the master branch -> assets folder of the Athavani repo